src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.ts

```diff
@@ -82,6 +83,17 @@ export class ApiInterceptorService implements HttpInterceptor {
                 }
               });
               break;
+            case 503:
+              this.router.navigate(['error'], {
+                state: {
+                  // TODO: change with response error message.
+                  message: $localize`The connection with monitor's daemons was compromised.`,
+                  header: $localize`Monitor connection error`,
+                  icon: 'fa fa-warning',
+                  source: 'forbidden'
+                }
+              });
+              break;
```

Comment:
I recently discovered that, when you enable/disable any mgr module from within the Ceph Dashboard (cluster - mgr modules, click "play/stop" button), the interface becomes locked/grayed out and a spinner appears during the mgr restart process. Maybe we can replicate that behaviour in here?